ui-particles-angular/projects/ui-policy-studio-angular/testing/ng-package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "$schema": "../../../../node_modules/ng-packagr/ng-package.schema.json",
+  "lib": {
+    "entryFile": "../src/public-testing-api.ts"
```

Comment:
It's a kind of hack 😁 
I didn't want to separate the testing part from the lib to keep it next to the source like the fixture with the model, sepc with compo, ext..

Testing folder here + the ng-package config is only used to create the `/testing` of the library (as proposed by angular material for example).
all the code to include here must be made public by `public-testing-api.ts`